configs/app.testing.json

```diff
@@ -1,15 +1,11 @@
 {
     "type": "gpii.appTestingWithTaskTray",
     "options": {
-        "gradeNames": ["fluid.component"],
-        "distributeOptions": {
-            "distributeDevPcpChannelConnector": {
-                "record": "gpii.app.dev.gpiiConnector",
-                "target": "{that gpiiConnector}.options.gradeNames"
-            }
-        }
+        "gradeNames": ["fluid.component"]
     },
     "mergeConfigs": [
-        "%gpii-app/configs/app.base.json"
+        "%gpii-app/configs/app.base.json",
+        "%flowManager/configs/gpii.flowManager.config.resetAtStart.base.json5",
```

Comment:
If we're only talking about DB content, this should not be needed.  I assume this is to reset the OS state?

Reply:
Just for my own understanding, I tracked it back through to the [flow manager](https://github.com/GPII/universal/blob/9f07ae63cd0a8b2027453dff5f33b70a45065e5a/gpii/node_modules/flowManager/src/FlowManager.js#L74) to confirm that this is [resetting the OS to the default settings](https://github.com/GPII/universal/blob/9f07ae63cd0a8b2027453dff5f33b70a45065e5a/gpii/node_modules/flowManager/src/FlowManager.js#L299).

Reply:
Right, this config makes the system to trigger a "Reset to STD" at gpii-app startup.
You can read a bit more about this functionality [here](https://github.com/GPII/universal/blob/master/documentation/ResetComputer.md).